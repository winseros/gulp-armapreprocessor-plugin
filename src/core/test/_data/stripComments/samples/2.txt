some text /* with block
// comment
data*/

and 

/*
more comments/*

and/*a bit more*/comments/* //
and some */
more*/
more*/
data

/* *
aa