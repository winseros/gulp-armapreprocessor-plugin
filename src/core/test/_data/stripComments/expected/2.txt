some text 



and 




comments

more*/
more*/
data


